LEAF_Nexus/sources/Export.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/*
+ * As a work of the United States government, this project is in the public domain within the United States.
+ */
+
+/*
+    This is an implementation of exportPDL.php, designed to work with the existing API handling system.
+
+*/
+
+namespace Orgchart;
+
+use App\Leaf\Db;
+
+class Export
+{
+    protected $db;
+    protected $login;
+
+    public function __construct($db, $login)
+    {
+        $this->db = $db;
+        $this->login = $login;
+    }
+
+    // Contents primarily copied from exportPDL.php
+    public function exportPDL() : array {
```

Comment:
I am not exactly sure what a PDL is, might be useful to have a phdoc for types and to inform what certain acronyms are.